.github/workflows/ci.yml

```diff
@@ -1,21 +1,15 @@
 name: CI
 
 # Runs build and test on:
-#   every push to main that has a change in a file not in the docs folder
-#   every pull request with main branch as the base that has a change in a file not in the docs folder
+#   every push to main
+#   every pull request with main branch as the base
 on:
   push:
     branches:
       - main
```

Comment:
@gksander @ryan-roemer I just noticed that when merging to `main`, it runs this CI workflow as well as the "Release" workflow.
Should we eliminate this CI workflow on `main`, so that merge-to-main only runs the Release workflow?

Reply:
In my experience, it's pretty common to run something like this even on push to `main`! I'm assuming to pick up any sort of "force push to main" scenarios, to make sure every commit to `main` definitely passes the expected tests.